scripts/gen_help_html.lua

```diff
@@ -1221,7 +1244,7 @@ function M.gen(help_dir, to_dir, include, commit, parser_path)
     local html, stats = gen_one(f, to_fname, not new_layout[helpfile], commit or '?', parser_path)
     tofile(to_fname, html)
     print(
-      ('generated (%-4s errors): %-15s => %s'):format(
+      ('generated (%-4s parse errors): %-15s => %s'):format(
```

Comment:
I intentionally removed "parse" from this earlier. It's over-explaining.
```suggestion
      ('generated (%-4s errors): %-15s => %s'):format(
```

Reply:
I see. When some files had invalid tags or other errors, I found it saying "generated (0 errors)", but I'm fine with reverting it.